### PR TITLE
PHP 7.4

### PR DESCRIPTION
**Pull request by sebastijansakac**

### CODE REVIEW

**Comment by tomazpu** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage decreased per file
===========================
- src/BackendService.php  -6
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/wirecard/paymentSDK-php/pullRequest?prid=5773317&bid=18896364)

**Comment by tomazpu** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ src/Entity/RiskInfo.php  2
+ src/Entity/TransactionDetails.php  1
+ src/Response/XmlResponseData.php  1
+ src/Entity/CustomFieldCollection.php  1
+ src/Helper/Enum.php  2
+ src/Entity/CustomField.php  2
+ src/Transaction/Transaction.php  1
         

Coverage decreased per file
===========================
- src/Entity/Device.php  -4
- src/Mapper/RequestMapper.php  -4
- src/Config/Config.php  -2
- src/Transaction/EpsTransaction.php  -10
- src/Entity/Browser.php  -2
- src/Helper/XmlBuilder.php  -10
- src/Entity/Redirect.php  -8
- src/TransactionService.php  -15
- src/Entity/AccountHolder.php  -1
- src/BackendService.php  -6
- src/Entity/Payload/NvpPayloadData.php  -8
- src/Entity/Mappable.php  -1
- src/Entity/Basket.php  -2
- src/Response/ResponseFactory.php  -9
- src/Entity/Address.php  -2
- src/Transaction/CustomFieldTransaction.php  -15
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/wirecard/paymentSDK-php/pullRequest?prid=5773317&bid=18896364)

**Comment by tomazpu** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage decreased per file
===========================
- src/TransactionService.php  -6
- src/TransactionService.php  -6
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/wirecard/paymentSDK-php/pullRequest?prid=5773317&bid=18896364)

**Comment by tomazpu** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage decreased per file
===========================
- src/TransactionService.php  -8
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/wirecard/paymentSDK-php/pullRequest?prid=5773317&bid=18896364)

File: test/TransactionServiceUTest.php [hunk 1]
```diff
@@ -48,18 +48,22 @@ class TransactionServiceUTest extends PHPUnit_Framework_TestCase
 
     private $shopSystemVersion;
 
+    private $config;
```
Comment (ozada) [inline test/TransactionServiceUTest.php:51]: Please add type hinting

File: test/TransactionServiceUTest.php [hunk 2]
```diff
@@ -48,18 +48,22 @@ class TransactionServiceUTest extends PHPUnit_Framework_TestCase
 
     private $shopSystemVersion;
```
Comment (ozada) [inline test/TransactionServiceUTest.php:49]: Please add type hinting

File: test/TransactionServiceUTest.php (hunk 1 above)
Comment (sebastijansakac) [reply test/TransactionServiceUTest.php:55]: Type hint added

File: test/TransactionServiceUTest.php (hunk 2 above)
Comment (sebastijansakac) [reply test/TransactionServiceUTest.php:50]: Type hint added

**Comment by tomazpu** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage decreased per file
===========================
- src/Transaction/CustomFieldTransaction.php  -10
- src/TransactionService.php  -8
- src/Response/ResponseFactory.php  -4
- src/BackendService.php  -21
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/wirecard/paymentSDK-php/pullRequest?prid=5773317&bid=18896364)

**Pull request merged**